src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,107 @@
+import {Observable} from 'rxjs';
+import {PeopleAdapter, PersonStatus} from '@webex/component-adapter-interfaces';
+
+export const USER_PRESENCE_UPDATE_EVENT = 'event:apheleia.subscription_update';
+
+/**
+ * The `PeopleSDKAdapter` is an implementation of the `PeopleAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a person.
+ *
+ * @export
+ * @class PeopleSDKAdapter
+ * @extends {PeopleAdapter}
+ */
+export default class PeopleSDKAdapter extends PeopleAdapter {
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @param {string} status
+   * @returns {PersonStatus.<String>}
```

Comment:
I'm not sure I understand what the type is supposed to be. Shouldn't this just be:
```suggestion
   * @returns {String} PersonStatus
```
since `PersonStatus` isn't an actual type